doc/http_methods.rst

```diff
@@ -0,0 +1,221 @@
+======================
+HTTP Methods in ODFUZZ
+======================
+
+
+Brief Overview of HTTP verb implementation
+------------------------------------------
+
+
+.. csv-table:: HTTP verb implementation
+   :header: "", "GET", "DELETE", "PUT", "POST" 
+   :widths: 30, 15, 15, 15, 15
+
+   "$inlinecount", "Yes", "No", "No", "No"
+   "$search", "Yes", "No", "No", "No"
+   "$top", "Yes", "No", "No", "No"
+   "$skip", "Yes", "No", "No", "No"
+   "$orderby", "Yes", "No", "No", "No"
+   "$expand", "Yes", "No", "No", "No"
+   "$filter", "Yes", "No", "No", "No"
+   "$format", "Yes", "No", "No", "No"
+   "body value", "Yes", "Yes", "Yes", "Yes"
+   "populated body", "No", "No", "Yes", "Yes"
+   "Addressing Single entities", "Yes", "Yes", "Yes", "No"
+   "Addressing Multiple entities", "Yes", "Yes", "No", "Yes"
+   "Synced URI and body values", "No", "No", "Yes", "No"
+
+**"body value"** refers to the fact that a value for the body is returned by *generate()* function. 
+**"populated body"** refers to the fact that the value contains data, and isnt an empty JSON parsed into a string.
+
+
+
+Introduction
+------------
+
+Odfuzz through the DirectBuilder class offers developers an interface to integrate its query creation functionalities into their project.
+As of ver. **0.14a2** the supported list of HTTP methods in DirectBuilder are **"GET", "DELETE", "PUT" and "POST"**. 
```

Comment:
You can delete the part "As of ver. **0.14a2** " since this information is in Git history anyway and the line "the supported list of HTTP methods in DirectBuilder are..." is enough :)

Reply:
ok, will do it myself :)